front-end/src/services/questionService.js

```diff
@@ -10,8 +10,8 @@ export const getUnitsByLevel = async (level) => {
   return data;
 };
 
-export const getTopicDataByUrl = async (params) => {
-  const { data } = await API.get('/api/questions/url', { params });
+export const getTopicDataByUrl = async (url) => {
```

Comment:
What if we need more params? Previous changes are more generic, so return it

Reply:
fix it